cmd/utils/flags.go

```diff
@@ -1938,7 +1965,7 @@ func MakeChain(ctx *cli.Context, stack *node.Node) (chain *core.BlockChain, chai
 	if ctx.GlobalBool(FakePoWFlag.Name) {
 		ethashConf.PowMode = ethash.ModeFake
 	}
-	engine = ethconfig.CreateConsensusEngine(stack, config, &ethconfig.Defaults, nil, false, chainDb)
+	engine = ethconfig.CreateConsensusEngine(stack, config, &ethconfig.Defaults, nil, false, false, chainDb)
```

Comment:
need to pass in `cfg.PrivateKeyForDataFieldSignature` to `CreateConsensusEngine()` instead of `false`

Reply:
This is a redundant parameter pass based on an old approach, so removing entirely, thanks